README.md

```diff
@@ -1 +1,51 @@
-# Leaderboard
\ No newline at end of file
+
+# Leaderboard
+
+> A microverse project to use APIs to reveive data from a server.
+
+
+## Built With
+
+- Html
+- Css
+- Javascript
+- VsCode
+- Webpack
+
+## Live Demo 
+
+[Live Demo Link](https://tugsem.github.io/Leaderboard/dist)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps:
+
+### Prerequisites
+
+-You need to have current Node version installed.
+
+### Setup
+
+-Copy the clone address or download as a zip into your local folder.
```

Comment:
- [ ] Please add a link to your repo and direction also how to `cd folder` and also how to run the localhost